apko.yaml

```
@@ -39,6 +45,8 @@ secfixes:
     - CVE-2023-28840
     - CVE-2023-28841
     - CVE-2023-28842
+  0.8.0-r1:
+    - GHSA-2h5h-59f5-c5x9
```

I missed this one 🤦 1,000 apologies!!!!
```suggestion
    - CVE-2023-30551
```